cmd/alertmanager/main.go

```diff
@@ -209,7 +209,8 @@ func run() int {
 		peerReconnectTimeout = kingpin.Flag("cluster.reconnect-timeout", "Length of time to attempt to reconnect to a lost peer.").Default(cluster.DefaultReconnectTimeout.String()).Duration()
 	)
 
-	promlogflag.AddFlags(kingpin.CommandLine, &promlogConfig)
+	a := kingpin.CommandLine.UsageWriter(os.Stdout)
+	promlogflag.AddFlags(a, &promlogConfig)
```

Comment:
can't we use kingpin.CommandLine here?

Reply:
Sorry I didn't understand what you mean, could you elaborate?